python/brayns/service/plugin.py

```diff
@@ -33,7 +33,7 @@ class Plugin(Enum):
     CIRCUIT_EXPLORER = 'braynsCircuitExplorer'
     ATLAS_EXPLORER = 'braynsAtlasExplorer'
     CYLINDRIC_CAMERA = 'braynsCylindricCamera'
-    CIRCUIT_INFO = 'braynsCircuitInfo'
+    #CIRCUIT_INFO = 'braynsCircuitInfo'
```

Comment:
Better leaving it uncommented so we don't forget to remove it when we will remove the plugin no ?

Reply:
Yes, I forgot to renable it. Addressed.